test/test_dropbox_unit.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
```

Comment:
I wish we have some test for `_refresh_access_token` which patch the return from server